sematic/resolvers/resource_requirements.py

```diff
@@ -277,14 +306,19 @@ class KubernetesResourceRequirements:
         The Kubernetes security context the job will run with. Note that this
         field will only be respected if ALLOW_CUSTOM_SECURITY_CONTEXTS has been
         enabled by your Sematic administrator.
+    host_path_mounts: List[KubernetesHostPathMount]
+        The "hostPath"-type configurations for volumes to mount on the pod to allow access
+        to the underlying nodes' file systems. More details can be found here:
+        https://kubernetes.io/docs/concepts/storage/volumes/#hostpath
     """
 
     node_selector: Dict[str, str] = field(default_factory=dict)
     requests: Dict[str, str] = field(default_factory=dict)
     secret_mounts: KubernetesSecretMount = field(default_factory=KubernetesSecretMount)
     tolerations: List[KubernetesToleration] = field(default_factory=list)
     mount_expanded_shared_memory: bool = field(default=False)
-    security_context: Optional[KubernetesSecurityContext] = None
+    security_context: Optional[KubernetesSecurityContext] = field(default=None)
+    host_path_mounts: List[KubernetesHostPathMount] = field(default_factory=list)
```

Comment:
i think these need to be deep-copied in the `clone()` below

Reply:
+1